app.test.js

```diff
@@ -57,4 +57,28 @@ describe('get /api',()=>{
     })
 })
 })
-    
\ No newline at end of file
+    
+describe('get /api/articles',()=>{
+    test('should return 200 and an array of article objects with the correct properties',()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body})=>{
+
+            expect(typeof body).toBe('object')
+            const { articles } = body;
+            expect(articles.length).toBe(13)
+            articles.forEach((article)=>{
+            expect(typeof article).toBe('object')
```

Comment:
again checking typeof on line 72 is redundant because article has to be a object to have properties on it